artemis-serialization/.gitignore

```diff
@@ -0,0 +1,2 @@
+save_temp.json
+save_temp.bin
```

Comment:
are these generated by a test or something?

Reply:
Yes, in JsonWorldSerializationManagerTest and KryoWorldSerializationManagerTest. Since they were never commited I guessed they should never be committed.